### PR TITLE
Added a brush for biomes

### PR DESCRIPTION
**Pull request by me4502**
Currently has pretty meh UX due to biomes not updating straight away. Arrangers may be needed to fix this, however.

Adds #1322

### CODE REVIEW

File: worldedit-core/src/main/resources/lang/strings.json [hunk 1]
```diff
@@ -29,7 +29,8 @@
     "worldedit.brush.operation.equip": "Set brush to {0}.",
     "worldedit.brush.none.equip": "Brush unbound from your current item.",
 
-    "worldedit.setbiome.changed": "Biomes were changed for approximately {0} blocks. You may have to rejoin your game (or close and reopen your world) to see a change.",
+    "worldedit.setbiome.changed": "Biomes were changed for approximately {0} blocks.",
+    "worldedit.setbiome.warning": "You may have to rejoin your game (or close and reopen your world) to see changes.",
```
Comment (octylFractal) [inline worldedit-core/src/main/resources/lang/strings.json:33]: While we're here:
```suggestion
    "worldedit.setbiome.warning": "You may have to re-join your game (or close and re-open your world) to see changes.",
```

File: worldedit-core/src/main/java/com/sk89q/worldedit/function/factory/Biome.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+package com.sk89q.worldedit.function.factory;
+
+import com.sk89q.worldedit.function.Contextual;
+import com.sk89q.worldedit.function.EditContext;
+import com.sk89q.worldedit.function.RegionFunction;
+import com.sk89q.worldedit.function.biome.BiomeReplace;
+import com.sk89q.worldedit.function.pattern.BiomePattern;
+
+public class Biome implements Contextual<RegionFunction> {
```
Comment (kashike) [inline worldedit-core/src/main/java/com/sk89q/worldedit/function/factory/Biome.java:28]: Maybe `SetBiome`?

Comment (wizjany) [reply worldedit-core/src/main/java/com/sk89q/worldedit/function/factory/Biome.java:28]: `ApplyBiome` would be more consistent with the other functions.

File: worldedit-core/src/main/java/com/sk89q/worldedit/command/BiomeCommands.java [hunk 3]
```diff
@@ -181,9 +181,11 @@ public void setBiome(Player player, LocalSession session, EditSession editSessio
         Operations.completeLegacy(visitor);
 
         player.printInfo(TranslatableComponent.of(
-                "worldedit.setbiome.changed",
-                TextComponent.of(visitor.getAffected())
-        ));
+            "worldedit.setbiome.changed",
+            TextComponent.of(visitor.getAffected())
+        )
```
Comment (wizjany) [inline worldedit-core/src/main/java/com/sk89q/worldedit/command/BiomeCommands.java:186]: indentation looks wonk

File: worldedit-core/src/main/java/com/sk89q/worldedit/command/BrushCommands.java [hunk 4]
```diff
@@ -433,6 +435,24 @@ public void snow(Player player, LocalSession localSession,
...
+                      @Arg(desc = "The shape of the region")
+                          RegionFactory shape,
+                      @Arg(desc = "The size of the brush", def = "5")
+                          double radius,
+                      @Arg(desc = "The biome type")
+                          BiomeType biomeType) throws WorldEditException {
+
+        setOperationBasedBrush(player, localSession, radius,
+            new ApplyRegion(new Biome(biomeType)), shape, "worldedit.brush.biome");
+        player.printInfo(TranslatableComponent.of("worldedit.setbiome.warning"));
```
Comment (wizjany) [inline worldedit-core/src/main/java/com/sk89q/worldedit/command/BrushCommands.java:453]: potentially this message should also be applied to `regen -b` and `paste/stack/move -b` (though it doesn't directly concern this PR but now that the message is its own key...)

**Pull request merged**